### PR TITLE
Landing page created. Resolves #187

### LINKED ISSUE #187: High-conversion standard landing page?
Reading https://unbounce.com/landing-page-examples/high-converting-landing-pages/ I think it would be nice to create a simple landing page.

The card pages would still be accessible from the nav, but the homepage would be simpler with the standard hero section.

I'm still trying to figure out what the best call-to-action in the hero section would be. Open to ideas about what would be most motivating to get site visitors to help families in need, one way or another. 

Currently we're rich in volunteers, poor in laptops & funding to help families, so perhaps something related to our biggest needs would be nice, but I'm open to whatever would make sense. Used laptop donations are probably the most urgent item, as we have space for a free community computer room, but our staff and all the families who come in for assistance are all taking turns sharing 2 laptops at the office 😅

### PR DESCRIPTION
**Pull request by LowLifeArcade**
I created a landing page with a hero section on top and a testimonial section below that. 🥳 

@audreyfeldroy here's a list of what remains:
- [x] Split up landing-page.js into separate files for the components. 
- [x] Convert media queries to use tailwind utility classes for breakpoints.
- [ ] Replace placeholder text with real copy.
- [ ] Change site homepage to be landing-page.js

Resolves #187

### CODE REVIEW

File: pages/landing-page.js [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+          <div className="card glass lg:card text-neutral-content">
+            <figure className="p-0">
+              <Image
+                alt="testimonial"
+                className="shadow-lg"
+                height={340}
+                layout="responsive"
+                src="/images/HealthWorkShop.jpg"
+                width={500}
+              />
```
Comment (RedFox0x20) [inline pages/landing-page.js:258]: A quick pointer regarding images, it's ideal to import the image as it automatically provides the correct width and height, ensuring the image looks as good as possible.

```js
import HealthWorkshopImage from '/images/HealthWorkshop.jpg';
...
src={HealthWorkshopImage}
```

File: pages/landing-page.js [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+              }
+            </p>
+
+            <button
+              className="btn btn-accent rounded-full shadow-lg"
+              onClick={() => router.push('/in-kind')}
+              type="button"
+            >
+              {'Help Today'}
+            </button>
```
Comment (RedFox0x20) [inline pages/landing-page.js:419]: Perhaps instead of a `button`, a `Link` from NextJS can be used instead.

**Review by RedFox0x20 — COMMENTED** [review]
@LowLifeArcade Thanks for the contributions, I can see that you have left plenty of notes of things that are still a work in progress and listed these things out, as such I have marked this PR as a Draft while it is worked on.

I'm liking the look of this page, it's very tidy in particular the testimonial section!

Just to add to your list of things to tweak, I believe it would be ideal to have the page share the same basic look and feel as the rest of the site. In particular I'm looking at the navbar which has a completely different look to it.

Also to avoid the `<style>` tag where possible, use of the Tailwind and DaisyUI classes is very much preferred and for almost every change made in the `<style>` can be done using `className`, this will likely occur naturally as elements are broken up into smaller components
> Split up landing-page.js into separate files for the components.

Finally in regards to the following, perhaps moving `index.js` to something more suitable such as `home.js` and subsequently moving `landing-page.js` to `index.js` would be a suitable option for this.
> Change site homepage to be landing-page.js

File: pages/landing-page.js [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+ * - `opacity` : "bg-opacity-`opacity`" | Example: `"30"` | [Tailwind opacity](https://tailwindcss.com/docs/background-opacity)
+ * - `heroHeight` : Use standard css (vh | px | %) :  Example: `"70vh"`
+ *
+ * @param {Object} props - bgImage, mainTextSize, opacity, title, titleSize
+ * @param {String} props.bgImage - Background image url
+ * @param {String} [props.heroHeight = '100vh'] Use standard css (vh | px | %) :  Example: `"70vh"`
+ * @param {String} [props.opacity = '60' ]"bg-opacity-`opacity`" | Example: `"30"` | [Tailwind opacity](https://tailwindcss.com/docs/background-opacity)
+ * @param {*} props.children - Children elements
+ * @returns {ReactElement} JSX
+ */
```
Comment (audreyfeldroy) [inline pages/landing-page.js:31]: This is really cool @LowLifeArcade!

File: pages/landing-page.js [hunk 4]
```diff
@@ -0,0 +1,437 @@
+// @ts-check
+// eslint-disable-next-line no-unused-vars
+import { ReactElement, useEffect, useState } from 'react'; // ReactElement is only imported for typechecking
+import Footer from '../components/Footer';
+import Head from 'next/head';
+import Image from 'next/image';
+import PropTypes from 'prop-types';
+import { useRouter } from 'next/router';
+
+// import { BsSearch } from 'react-icons/bs';
```
Comment (audreyfeldroy) [inline pages/landing-page.js:10]: Hey, I'm curious why you decided to put this in, then comment it out? I vaguely remember it being in the navbar?

File: pages/landing-page.js [hunk 5]
```diff
@@ -0,0 +1,437 @@
...
+        <title>{'Help a Family in Need'}</title>
+        <meta content={'Help a Family in Need'} name="description" />
+        <link href="https://www.margaritahumanitarian.org/" rel="canonical" />
+        <link href="/favicon.ico" rel="icon" />
+      </Head>
+      <LandingPageNav fadeInSpeed="1.5s" />
+      <main className="body">
+        <LargeHeroSection
+          bgImage="/images/HotMealDay.jpg"
+          heroHeight="85vh"
```
Comment (audreyfeldroy) [inline pages/landing-page.js:398]:
```suggestion
```

Let's try removing this to let `heroHeight be the default value from line 36.

File: pages/landing-page.js (hunk 2 above)
Comment (audreyfeldroy) [reply pages/landing-page.js:419]: That's a good idea. @LowLifeArcade check out the Next.js `Link` docs at https://nextjs.org/docs/api-reference/next/link and let us know if you need any help figuring it out 😄

**Review by audreyfeldroy — CHANGES_REQUESTED** [review]
This is great @LowLifeArcade, thanks for submitting it! Try out the "Commit Suggestion" button to see what it does, and we can continue to work on this PR together over screenshare. 😄

File: pages/landing-page.js (hunk 4 above)
Comment (RedFox0x20) [reply pages/landing-page.js:10]: That element has been commented out, in addition to a few other elements in the navbar which here is a copy of the original code with redactions. Depending on the decisions made with the navbar here, I may modify the original to make it more suitable to these broader cases where elements may want to be added/remove per page.

**Comment by LowLifeArcade** [comment]
> @LowLifeArcade Thanks for the contributions, I can see that you have left plenty of notes of things that are still a work in progress and listed these things out, as such I have marked this PR as a Draft while it is worked on.

Yeah!

> Just to add to your list of things to tweak, I believe it would be ideal to have the page share the same basic look and feel as the rest of the site. In particular I'm looking at the navbar which has a completely different look to it.

I hear you on that. The navbar is an interesting point as I am trying to figure out how it fits in. As per the referred [blog post](https://unbounce.com/landing-page-examples/high-converting-landing-pages/) there's a push to almost omit the whole thing. @audreyfeldroy had stated she wanted to keep it in, so I modified it to meet the recommendations. Stylistically ,going with the blog post, it made sense that it was less observable and to really only appear as branding in my mind, so I went with an opaque bar with minimal content. Of course we're all still in conversation about these things, so let's keep talking and sharing thoughts.

> Also to avoid the `<style>` tag where possible, use of the Tailwind and DaisyUI classes is very much preferred and for almost every change made in the `<style>` can be done using `className`, this will likely occur naturally as elements are broken up into smaller components

I took this into account as well as with a conversation with @audreyfeldroy and refactored the code using tailwind's utility classes. 

> > Split up landing-page.js into separate files for the components.

Definitely!

> Finally in regards to the following, perhaps moving `index.js` to something more suitable such as `home.js` and subsequently moving `landing-page.js` to `index.js` would be a suitable option for this.
> > Change site homepage to be landing-page.js

I believe @audreyfeldroy has a plan or idea there.

**Comment by audreyfeldroy** [comment]
Merging this so we can iterate further during Hacktoberfest, breaking remaining to-do's into small, easy ones for beginners to contribute to. The landing page is still only hidden at /landing-page so it shouldn't affect the rest of the site. Thanks for all your work here @LowLifeArcade and for your thorough and thoughtful reviewing @RedFox0x20 🙏

**Review by audreyfeldroy — APPROVED** [review]

**Pull request merged**